.github/workflows/create-quantic-package.yml

```diff
@@ -60,8 +60,8 @@ jobs:
       - name: Connect to DevHub organization
         working-directory: ./packages/quantic
         run: |
-          echo $SFDX_AUTH_JWT_KEY > server.key
-          sfdx force:auth:jwt:grant --clientid $SFDX_AUTH_JWT_CLIENT_ID --jwtkeyfile server.key --username $SFDX_AUTH_JWT_USERNAME --setdefaultdevhubusername
+          echo "$SFDX_AUTH_JWT_KEY" > server.key
```

Comment:
Yes according to the similar action we have in [salesforceintegration](https://github.com/coveo/salesforceintegration/blob/7c70f620eac55176dbeea9ccab99cba4260a4d53/.github/actions/release-1GP/action.yml#L11) this is more adequate.